.eslintrc

```diff
@@ -1,3 +1,5 @@
+extends: eslint:recommended
```

Comment:
Hmm ... I started to agree with you, but I also like following conventions over configurations. 

I.e., if the ESLint team (and community) changes their rules in a way that breaks our front-style checks, it's a trigger to us to decide - either follow the ESLint community convention or keep our existing convention.